src/Compilers/Core/Portable/MethodImplExtensions.cs

```diff
@@ -0,0 +1,26 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Reflection;
+using System.Runtime.CompilerServices;
+
+namespace Microsoft.CodeAnalysis;
+
+internal static class MethodImplOptionsExtensions
```

Comment:
It looks like the names of the static classes were swapped. This one should be `MethodImplAttributeExtensions`